GVRf/Framework/framework/src/main/java/org/gearvrf/GVRConfigurationManager.java

```diff
@@ -142,6 +142,16 @@ void setDockListenerRequired(boolean value){
      */
     public abstract boolean isHmtConnected();
 
+    /**
```

Comment:
Methods unused; please remove. In any case only the Oculus backend is supposed to support OVR_Multiview at the moment so this needs to be implemented to return always false for all other backend.